core/src/config.rs

```diff
@@ -54,10 +55,10 @@ pub struct HostImportDescriptor<WB: WasmBackend> {
 
 pub struct MModuleConfig<WB: WasmBackend> {
     /// Import object that will be used in module instantiation process.
-    pub raw_imports: HashMap<String, RawImportCreator<WB>>,
+    pub raw_imports: HashMap<u32, HashMap<String, RawImportCreator<WB>>>,
```

Comment:
I think it'd be better to introduce a new type for api_version, it'll make code more readable.